pallets/vc-management/src/lib.rs

```diff
@@ -368,5 +376,42 @@ pub mod pallet {
 			Self::deposit_event(Event::SchemaRevoked { account: sender, shard, index });
 			Ok(().into())
 		}
+
+		/// ---------------------------------------------------
+		/// The following extrinsics are supposed to be called by Sudo/Concuil only
+		/// This is the temporary function for manual operation of VCRegistry storage
+		/// ---------------------------------------------------
+		#[pallet::call_index(10)]
+		#[pallet::weight(195_000_000)]
+		pub fn add_vc(
+			origin: OriginFor<T>,
+			index: VCIndex,
+			subject: T::AccountId,
+			assertion: Assertion,
+			hash: H256,
+		) -> DispatchResultWithPostInfo {
+			T::SetAdminOrigin::ensure_origin(origin)?;
+			ensure!(!VCRegistry::<T>::contains_key(index), Error::<T>::VCAlreadyExists);
+			VCRegistry::<T>::insert(
+				index,
+				VCContext::<T>::new(subject.clone(), assertion.clone(), hash),
+			);
+			Self::deposit_event(Event::VCRegistryManualAdded {
+				account: subject,
+				assertion,
+				index,
+			});
+			Ok(().into())
+		}
+
+		#[pallet::call_index(11)]
+		#[pallet::weight(195_000_000)]
+		pub fn remove_vc(origin: OriginFor<T>, index: VCIndex) -> DispatchResultWithPostInfo {
+			T::SetAdminOrigin::ensure_origin(origin)?;
+			let _ = VCRegistry::<T>::get(index).ok_or(Error::<T>::VCNotExist)?;
+			VCRegistry::<T>::remove(index);
+			Self::deposit_event(Event::VCRegistryManualRemoved { index });
+			Ok(().into())
+		}
```

Comment:
let's have another extrinsic to remove_all -> `clear()` is enough

Reply:
added